codebase2/codebase-sqlite/unison-codebase-sqlite.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.34.4.
+-- This file has been generated from package.yaml by hpack version 0.34.7.
```

Comment:
Can `nix` give you a stack with hpack 0.34.4? (: nix question

Reply:
Oh yes, I will downgrade hpack.